From weiyi.huang@gmail.com  Mon Oct 26 15:08:05 2009
From: Huang Weiyi <weiyi.huang@gmail.com>
Date: Thu, 17 Sep 2009 21:25:37 +0800
Subject: Staging: rtl8187su: remove unused #include <linux/version.h>
To: gregkh@suse.de
Cc: devel@driverdev.osuosl.org, Huang Weiyi <weiyi.huang@gmail.com>
Message-ID: <1253193937-5160-1-git-send-email-weiyi.huang@gmail.com>


Remove unused #include <linux/version.h>('s) in
  drivers/staging/rtl8192su/ieee80211/ieee80211_rx.c
  drivers/staging/rtl8192su/ieee80211/ieee80211_tx.c
  drivers/staging/rtl8192su/ieee80211/ieee80211_wx.c

Signed-off-by: Huang Weiyi <weiyi.huang@gmail.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/rtl8192su/ieee80211/ieee80211_rx.c |    1 -
 drivers/staging/rtl8192su/ieee80211/ieee80211_tx.c |    1 -
 drivers/staging/rtl8192su/ieee80211/ieee80211_wx.c |    1 -
 3 files changed, 3 deletions(-)

--- a/drivers/staging/rtl8192su/ieee80211/ieee80211_rx.c
+++ b/drivers/staging/rtl8192su/ieee80211/ieee80211_rx.c
@@ -37,7 +37,6 @@
 #include <linux/slab.h>
 #include <linux/tcp.h>
 #include <linux/types.h>
-#include <linux/version.h>
 #include <linux/wireless.h>
 #include <linux/etherdevice.h>
 #include <asm/uaccess.h>
--- a/drivers/staging/rtl8192su/ieee80211/ieee80211_tx.c
+++ b/drivers/staging/rtl8192su/ieee80211/ieee80211_tx.c
@@ -47,7 +47,6 @@
 #include <linux/slab.h>
 #include <linux/tcp.h>
 #include <linux/types.h>
-#include <linux/version.h>
 #include <linux/wireless.h>
 #include <linux/etherdevice.h>
 #include <asm/uaccess.h>
--- a/drivers/staging/rtl8192su/ieee80211/ieee80211_wx.c
+++ b/drivers/staging/rtl8192su/ieee80211/ieee80211_wx.c
@@ -30,7 +30,6 @@
 
 ******************************************************************************/
 #include <linux/wireless.h>
-#include <linux/version.h>
 #include <linux/kmod.h>
 #include <linux/module.h>
 
